llvm/include/llvm/IR/IntrinsicsDPU.td

```diff
@@ -11,8 +11,12 @@
 //
 //===----------------------------------------------------------------------===//
 
+def llvm_mram_ptr_ty : LLVMQualPointerType<llvm_i8_ty, 255>;
```

Comment:
We have a number of occurrences of this hardcoded `255`. Is it possible to abstract it somehow?

Reply:
I agree, but it seems hard to have it another way in the ``.td`` file. The only way that come to my mind is using autogen to generate this ``.td``.... Does not feel that good...

Reply:
Could we generate a separate `.td` file and include it in this file?